### PR TITLE
Add truncated GCM mesh back in

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR:
 - Adds the truncated GCM mesh capability back in
 - Changes that test to run `LES_driven_SCM` (cc @trontrytel, @ilopezgp)
 - Adds a doc string snippet to `print_repl_script`

### CODE REVIEW

File: driver/common_spaces.jl [hunk 1]
```diff
@@ -30,10 +30,9 @@ function make_horizontal_space(mesh, quad)
     return space
 end
 
-function make_hybrid_spaces(h_space, z_max, z_elem, z_stretch)
-    FT = eltype(z_max)
-    z_domain = CC.Domains.IntervalDomain(CCG.ZPoint(zero(z_max)), CCG.ZPoint(z_max); boundary_tags = (:bottom, :top))
-    z_mesh = CC.Meshes.IntervalMesh(z_domain, z_stretch; nelems = z_elem)
+function make_hybrid_spaces(h_space, z_mesh, ::Type{FT}) where {FT}
+    # z_domain = CC.Domains.IntervalDomain(CCG.ZPoint(zero(z_max)), CCG.ZPoint(z_max); boundary_tags = (:bottom, :top))
+    # z_mesh = CC.Meshes.IntervalMesh(z_domain, z_stretch; nelems = z_elem)
```
Comment (ilopezgp) [inline driver/common_spaces.jl:35]: Do we want to keep this?

**Review by ilopezgp — APPROVED** [review]

File: driver/common_spaces.jl (hunk 1 above)
Comment (charleskawczynski) [reply driver/common_spaces.jl:35]: Oh, whoops, no

**Review by trontrytel — APPROVED** [review]
LGTM!

**Comment by ilopezgp** [comment]
bors try

**Comment by trontrytel** [comment]
I think there is something wrong with this one since it breaks the CI and buildkite. 

The alternative that I wrote (https://github.com/CliMA/TurbulenceConvection.jl/pull/1232) seems to work in the CI, but is hit by some packages issues on buildkite

**Comment by ilopezgp** [comment]
```
type LinRange has no field z
```

**Comment by charleskawczynski** [comment]
Fixing…

**Pull request merged**